src/containers/Tasks/Tasks.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+// import axios from '../../axios-tasks';
+import { connect } from 'react-redux';
+import * as actionTypes from '../../store/actions/index'
+import { Redirect } from 'react-router-dom';
+
+import Task from './Task/Task';
+
+// import '../../assets/semantic/semantic.min.css';
+// import { Dimmer, Loader, Image, Segment } from 'semantic-ui-react';
+
+
+class Tasks extends Component {
+    state = {
+        fetchedTasks: [],
+        loading: false
+    };
+
+    componentDidMount = () => {
+        console.log('componentDidMount');
+        if(!this.props.taskList.length){
+            this.props.onInitTasks(this.props.token);
+            this.setState({ loading: true });
+        }
+    };
+
+    deleteTaskHandler = (taskId) => {
+        console.log(taskId);
+    };
+
+    render() {
+        let authRedirect = null;
+        if(this.props.token === null){
+            authRedirect = <Redirect to="/" />
+        }
+        let allTasks = <p>No task found! Try adding one.</p>
+        if(this.props.taskList.length){
+            allTasks = this.props.taskList.map(fetchedTask => (
+                <Task
+                    title={fetchedTask.title}
```

Comment:
do not use seperate prop as fields use one single obj as props example `task={fetchedTask}`